packages/atlas/src/components/_navigation/SidenavBase/SidenavBase.styles.ts

```diff
@@ -189,30 +129,29 @@ export const LegalLinksWrapper = styled.span`
 
 export const LegalLink = styled(Link)`
   text-decoration: none;
-  font: ${cVar('typographyDesktopT200Strong')};
-  letter-spacing: ${cVar('typographyDesktopT200StrongLetterSpacing')};
-  text-transform: ${cVar('typographyDesktopT200StrongTextTransform')};
+  color: ${cVar('colorTextMuted')} !important;
 
   &:hover {
-    color: ${oldColors.gray[400]};
+    color: ${cVar('colorTextMuted')};
   }
 
   &:focus {
-    color: ${oldColors.gray[400]};
-  }
-
-  &:active {
-    color: ${oldColors.gray[500]};
+    color: ${cVar('colorTextMuted')};
   }
 `
 
 export const StyledHamburgerButton = styled(HamburgerButton)`
   position: fixed;
+  margin-left: ${sizes(3)};
   z-index: ${zIndex.sideNav};
-  left: ${sizes(3)};
-  top: ${sizes(2)};
-
+  margin-top: ${sizes(3)};
   ${media.md} {
-    top: ${sizes(4)};
+    margin-top: ${sizes(4)};
```

Comment:
1. What's the reason behind the change from `top/left` to margins?
2. On mobile, the hamburger has wrong position still, it was correct before the change

Reply:
1. I don't understand. The hamburger button still has `position: fixed` so what difference does using margins make?
2. Like I said previously, this positioning was correct before your changes so if you revert your changes, everything should work fine. If you need help with measuring margins in Figma, let me know and I can help you with that